### ISSUE TITLE
Add links to Guides and Get Started page in Footer

### ISSUE THREAD

**Issue by SteveSamJacob19** [OpenLiberty/openliberty.io#3808]
The footer has to be updated by adding the links to Guides and Get Started page to maintain consistency with the Header. This does not have much user impact and hence can be considered low priority

**Comment by pranay911** [comment]
@SteveSamJacob19 Can you please assign this issue to me ?

**Comment by SteveSamJacob19** [comment]
@pranay911 Sure

**Comment by pranay911** [comment]
@SteveSamJacob19 Can you please send me the link ?

**Comment by SteveSamJacob19** [comment]
The header contains the links to "Guides" and "Get Started" pages, you can obtain it from there and just add it to the footer like how the "Docs", "Blogs" and "Support" are in the footer

**Comment by SteveSamJacob19** [comment]
@pranay911 Whats the status on this?

**Comment by JavierABarreto** [comment]
Hello @SteveSamJacob19, can you assign this issue to me? I would like to contribute

**Comment by SteveSamJacob19** [comment]
@JavierABarreto Sure, please do go through our wiki on how to contribute before creating a pull request

**Comment by JavierABarreto** [comment]
Hi @SteveSamJacob19, following the How to Contribute guide I tried to push the changes into the repo but it returned me an error:
<img width="791" alt="image" src="https://github.com/user-attachments/assets/ea616aa3-7d0b-4aa0-9975-83c28a7932e0">

Do I need permission to perform a push? Or it is better if I do a fork and create the PR from there?

**Comment by JavierABarreto** [comment]
Oh, never mind this ^, finally got to push the changes... sorry

**Comment by SteveSamJacob19** [comment]
Hi @JavierABarreto are you able to push your changes to your branch now?

**Comment by JavierABarreto** [comment]
Hi @SteveSamJacob19, I tried to push my changes but it still gives me an error:
<img width="559" alt="image" src="https://github.com/user-attachments/assets/56bc4af7-1753-45d5-8bc9-634cf181a72c">

**Comment by SteveSamJacob19** [comment]
Hi @JavierABarreto, Can you tell me the exact steps you've taken and to which branch you're pushing the changes to?

**Comment by JavierABarreto** [comment]
Hi @SteveSamJacob19, yes what I did was:
1. Clone the repository.
2. Set up the environment.
3. Created a new branch from main in my local called "adding-links-in-footer".
4. After making my changes, I tried to push my changes with the command ```git push origin adding-links-in-footer```

**Comment by SteveSamJacob19** [comment]
Did you clone the branch using SSH? If not can you try that?

**Comment by JavierABarreto** [comment]
Hi @SteveSamJacob19, I cloned the branch again this time using SSH, created the branch adding-links-in-foote again, and I still can't push my changes.

**Comment by SteveSamJacob19** [comment]
Ok, can you fork the Repo and push changes?

**Comment by JavierABarreto** [comment]
Hi @SteveSamJacob19, when I create the PR to draft, id adds some commits and files that aren't mine, is it because draft is not up to date with prod?

**Comment by SteveSamJacob19** [comment]
Hi @JavierABarreto , you have to rebase your branch to draft and then create a PR

**Comment by JavierABarreto** [comment]
Hi @SteveSamJacob19, I created the branch from prod, rebased with draft, added my changes and then created the PR and it still has some commits/files that aren't mine. I will work on that and on solving conflicts.

**Comment by SteveSamJacob19** [comment]
Hi @JavierABarreto, Can you create the PR anyways, ill look into it

**Comment by SteveSamJacob19** [comment]
@JavierABarreto I think you will have to fetch our repo upstream to ur forked repo and then "rebase" to draft and not merge draft.

**Comment by JavierABarreto** [comment]
That's actually what I did, gotta do it again to make sure that I didn't missed any step while "Rebasing"

**Comment by JavierABarreto** [comment]
Yup, it seems that I had to skip some commits while rebasing draft lol, there we go.

**Comment by JavierABarreto** [comment]
Hi @SteveSamJacob19, while reviewing the changes on the draft view I noticed that there's this small detail where the div of the links do this:
<img width="903" alt="image" src="https://github.com/user-attachments/assets/2d9ee70c-d6b1-43b1-9407-147fad4752a3">

My question is, do I change it? and if I do, may I modify [this media](https://github.com/OpenLiberty/openliberty.io/blob/78bc59d762d9aa58aa704d6e4eb0437b0c4e80c4/src/main/content/_assets/css/openliberty.scss#L587C1-L587C31) line to 910px? Modifying this media line to 910px would make this look better.

**Comment by SteveSamJacob19** [comment]
@JavierABarreto I will have to discuss on this and get back to you

**Comment by JavierABarreto** [comment]
Hi @SteveSamJacob19, then I do the PR to staging? or I wait for your answer?

**Comment by SteveSamJacob19** [comment]
Hi @JavierABarreto you can continue with the change into draft and only then move to staging cause it does look odd

**Comment by SteveSamJacob19** [comment]
Hi @JavierABarreto please add me as a reviewer whenever you open a  PR and also ensure that its rebased. I have done both this time but ensure the same when ur create a PR to staging

**Comment by JavierABarreto** [comment]
> Hi @JavierABarreto please add me as a reviewer whenever you open a PR and also ensure that its rebased. I have done both this time but ensure the same when ur create a PR to staging

Hi @SteveSamJacob19, I have tried before to add you as  a reviewer but it doesn't let me (it seems that I lack perms on adding who can review my PRs)

**Comment by JavierABarreto** [comment]
Hello @SteveSamJacob19, I created the PR to staging and it doesn't let me add anyone as a reviewer

**Comment by SteveSamJacob19** [comment]
@JavierABarreto Thats fine, i will take a look into the PR as soon as im free and push it if its correct

**Comment by SteveSamJacob19** [comment]
@JavierABarreto pls check the above PR opened and verify ur changes in it and let me know

**Comment by JavierABarreto** [comment]
@SteveSamJacob19 Looks good, all my changes are in there! 👍
